Eliminate warnings from selectors that are referenced before they are defined

--- HistoryController.m.orig	2023-02-04 19:03:15.000000000 -0500
+++ HistoryController.m	2023-02-04 19:03:22.000000000 -0500
@@ -37,6 +37,7 @@
 - (void) HC_setSortColumn: (NSString *) newSortColumn;
 - (NSImage *) HC_sortDescendingToImage;
 - (void) HC_setDateFormat;
+- (void) clearSheetDidEnd:(NSAlert *)alert returnCode:(NSInteger)returnCode contextInfo:(void *)contextInfo;
 
 @end
 
--- GameController.m.orig	2023-02-04 19:02:04.000000000 -0500
+++ GameController.m	2023-02-04 19:05:00.000000000 -0500
@@ -36,6 +36,12 @@
 @interface GameController (PrivateMethods)
 
 - (void) GC_startGame;
+- (void) startGameSheetDidEnd: (NSWindow *) sheet returnCode: (int) returnCode
+                  contextInfo: (void *) contextInfo;
+- (void) windowCloseSheetDidEnd: (NSWindow *) sheet returnCode: (int) returnCode
+                        contextInfo: (void *) contextInfo;
+- (void) applicationTerminateSheetDidEnd: (NSWindow *) sheet returnCode: (int) returnCode
+                             contextInfo: (void *) contextInfo;
 
 @end
 
